internal/build/lifecycle_execution_test.go

```diff
@@ -183,6 +183,37 @@ func testLifecycleExecution(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
+		when("Run with workspace dir", func() {
+			it("succeeds", func() {
+				opts := build.LifecycleOptions{
+					Publish:      false,
+					ClearCache:   false,
+					RunImage:     "test",
+					Image:        imageName,
+					Builder:      fakeBuilder,
+					TrustBuilder: false,
+					Workspace:    "app",
+					UseCreator:   true,
+				}
+
+				lifecycle, err := build.NewLifecycleExecution(logger, docker, opts)
+				h.AssertNil(t, err)
+
+				err = lifecycle.Run(context.Background(), func(execution *build.LifecycleExecution) build.PhaseFactory {
+					return fakePhaseFactory
+				})
+				h.AssertNil(t, err)
+
+				h.AssertEq(t, len(fakePhaseFactory.NewCalledWithProvider), 1)
+
+				for _, entry := range fakePhaseFactory.NewCalledWithProvider {
+					if entry.Name() == "creator" {
+						h.AssertSliceContains(t, entry.ContainerConfig().Cmd, "/some/image")
+					}
+				}
```

Comment:
Is there any way to verify in the phase creator that the mount path was returned correctly? This assertion doesn't seem super relevant to the change.